### PR TITLE
Add externalSetMarkChain to portmap config

### PR DESCRIPTION
**Pull request by champtar**
Looking at the online portmap doc, this seems to be
the recommended configuration for k8s + portmap
https://www.cni.dev/plugins/current/meta/portmap/

### CODE REVIEW

**Comment by Sea-n** [comment]
Hi @champtar,

Thanks for fixing this!
But in this project, we limit every PR to only touch one language.

Could you make this PR to fix only `content/en` file first? After the PR merged, we can open 4 PR for each language or let others do it.

Feel free to `/unhold` when it's done.
/hold

**Comment by Sea-n** [comment]
Let me remove redundant language labels for you.
/remove-language id ko pt zh

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 1]
```diff
@@ -87,7 +87,8 @@ For example:
     },
     {
       "type": "portmap",
-      "capabilities": {"portMappings": true}
+      "capabilities": {"portMappings": true},
+      "externalSetMarkChain": "KUBE-MARK-MASQ"
```
Comment (annajung) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:91]: I am not sure if this is necessary, hoping someone from @kubernetes/sig-network-pr-reviews can take a look

**Comment by reylejano** [comment]
Hi @kubernetes/sig-network-pr-reviews , please take a look and provide a technical review

**Comment by Sea-n** [comment]
It's been a while, ping the SIG again: @kubernetes/sig-network-pr-reviews

**Comment by aojea** [comment]
/assign @danwinship 

he has a KEP going on about the usage of the iptables chains, he is the best person to review this

**Comment by aojea** [comment]
however, after reading the whole doc, it seems it needs to update more sections

**Comment by sftim** [comment]
@aojea: even if the whole doc is not up-to-date, does this PR make things better or worse?

**Comment by aojea** [comment]
> @aojea: even if the whole doc is not up-to-date, does this PR make things better or worse?

/lgtm

it makes them better IMHO

**Comment by tengqm** [comment]
/approve
Leaving further tweaks to future work.

**Pull request merged**